### DISCUSSION TITLE
Editor - hitsound volume should be inherited from the previous object

### DISCUSSION THREAD

**Discussion by Digitalfear117** [ppy/osu#28290] [Ideas] [answered]
![A preview of a new circle being placed on lazer that uses the default HS volume](https://github.com/ppy/osu/assets/28713262/be0ca54f-106a-4527-8b8a-5e4de1923740)

Whenever I place a new circle on my map, the hitsound volume gets set back to the default of 100%, which is really annoying since I constantly have to change the hit sound volume to the previous one for every placed object…. I think it should just inherit the previous volume unless the user goes out of their way to change it. 

This seems similar to https://github.com/ppy/osu/discussions/18725, where the sample set is retained, but it would be nice if the volume % was retained too!

**Comment by pha-Z** [comment, marked as answer]
im trying out lazer myself right now and this issue seems to be resolved, with objects inheriting volume from prior objects

**Reply by peppy** [reply]
Yep, it was since fixed 👍
